include/config.h

```diff
@@ -1,6 +1,21 @@
 #ifndef GUARD_CONFIG_H
 #define GUARD_CONFIG_H
 
+// In the Generation 3 games, Asserts were used in various debug builds.
```

Comment:
We should add a short "call to action" like this to the beginning of this comment block.

"To enable print debugging, uncomment "#define NDEBUG".  This allows the various AGBPrint functions to be used. (See include/gba/isagbprint.h).  Some emulators support a debug console window. (And maybe call out no$gba's special function)